sovrin_common/exceptions.py

```diff
@@ -37,3 +37,8 @@ class LinkNotReady(RuntimeError):
 
 class NotConnectedToNetwork(RuntimeError):
     pass
+
+
+class SovrinNotAvailable(Exception):
```

Comment:
I understand the intent behind this exception is that it should be raise when client cannot talk to atleast f+1 nodes, right? Can we have more specific exceptions, a client might have a firewall which blocks him from accessing some nodes or something like that